### PR TITLE
Story/fba slider threshold

### PR DESCRIPTION
**Pull request by conbrad**
- Fire zone advisory endpoint (`/fire-zone-areas`) performs `hfi`/`fire_zones` postgis intersection lookup on the fly since we simplified the fire zones
- Adds a threshold slider on the frontend to paint advisory layer based on the dynamic hfi area % threshold number

### CODE REVIEW

**Review by Sybrand — COMMENTED** [review]
l.g.t.m for now - will re-look once my pr is in and it's using the new zone table.

File: api/app/routers/fba.py [hunk 1]
```diff
@@ -31,10 +31,17 @@ async def get_all_fire_centers(_=Depends(authentication_required)):
...
-            zones.append(FireZoneArea(mof_fire_zone_id=advisory.mof_fire_zone_id,
-                                      elevated_hfi_area=advisory.elevated_hfi_area,
-                                      elevated_hfi_percentage=advisory.elevated_hfi_percentage))
+        rows = await get_hfi_area_percentages(session, for_date)
+
+        # Fetch rows.
+        for row in rows:
+            zone_area = row.zone_area
+            hfi_area = row.hfi_area
+            print(f'{row.mof_fire_zone_name}:{hfi_area}/{zone_area}={hfi_area/zone_area*100}%')
```
Comment (andrea-williams) [inline api/app/routers/fba.py:41]: I don't think we need this print statement?

**Review by andrea-williams — APPROVED** [review]
I just reviewed the code, since there's no dev route to test it out and I'm feeling too lazy to pull and run it locally.

Aside from an extra print statement in routers/fba.py that I don't think should be needed, LGTM

File: api/app/routers/fba.py [hunk 2]
```diff
@@ -1,23 +1,65 @@
...
+        key = f'/vsis3/{bucket}/sfms/uploads/forecast/{run_date.isoformat()}/hfi{for_date_string}.tif'
+        with tempfile.TemporaryDirectory() as temp_dir:
+            temp_filename = os.path.join(temp_dir, 'classified.tif')
+            classify_hfi(key, temp_filename)
+            geojson = polygonize(temp_filename)
+            for feature in geojson.get('features', {}):
+                properties = feature.get('properties', {})
+                hfi = properties.get('hfi', None)
+                if hfi == 1:
+                    hfi = '4000 > hfi < 10000'
```
Comment (andrea-williams) [inline api/app/routers/fba.py:51]:
```suggestion
                    hfi = '4000 < hfi < 10000'
```

**Comment by Sybrand** [comment]
This PR superseded by https://github.com/bcgov/wps/pull/2256

**Pull request closed without merging**